packages/network-of-terms-catalog/README.md

```diff
@@ -0,0 +1,38 @@
+# Network of Terms Catalog
+
+This is the catalog of datasets that you can query through the
+[Network of Terms API](https://github.com/netwerk-digitaal-erfgoed/network-of-terms-api).
+
+The [catalog](catalog/) directory contains all datasets.
+Each is described in the [Schema.org](https://schema.org) ontology.
+
+The [queries](catalog/queries/) directory contains SPARQL queries for retrieving terms from the datasets.
+
+## Installation
+
+This package is [available on NPM](https://www.npmjs.com/package/@netwerk-digitaal-erfgoed/network-of-terms-catalog):
+
+```
+npm add @netwerk-digitaal-erfgoed/network-of-terms-catalog
+```
+
+## Contributing
+
+### Adding a dataset
+
+* Create a `your-dataset.jsonld` file in the `catalog/` directory and add a description.
+* Create a `your-dataset.rq` in the `queries/` directory to hold the SPARQL query.
```

Comment:
Perhaps we should make a distinction between the search and lookup queries here?